### ISSUE TITLE
Further considerations on fall damage

### ISSUE THREAD

**Issue by Montimorency** [CleverRaven/Cataclysm-DDA#72704]
### Is your feature request related to a problem? Please describe.

Fall damage changes in #72615 may not go far enough. 

### Solution you would like.

The benchmark scenario was an 87.5 kg wasp falling one z-level onto the player character. Do we have ready examples of 87.5 kg objects falling on people from a few meters? We do, of course, but in particular I would draw attention to pro wrestling. I am not a viewer, but as I understand it nearly every match features large wrestlers - often larger than 100 kg when male - deliberately leaping onto often prone or supine people from the ring posts or ropes. These techniques are referred to as "diving" or "flying" attacks. 

![800px-Undertaker_oldschool_2](https://github.com/CleverRaven/Cataclysm-DDA/assets/151707610/a7710f49-216c-4254-b7bd-bdd929250e94)
![ezgif-1-ded9df603f](https://github.com/CleverRaven/Cataclysm-DDA/assets/151707610/b04beac7-fb19-4980-a0fd-b9baa6e3882d)

It is absolutely possible to sustain injuries in pro wrestling or in any contact sport, even serious ones, such as broken bones or worse, but these are uncommon, and the norm is that wrestlers will continue to perform immediately after suffering such attacks. Acute ill effects must be no more than some pain. 

Mind you, these are actual leaps and dives, not just gravitational collisions. And from what I can tell, the risk of injury is much higher for those performing diving attacks than for those receiving them. 

While not studying American professional wrestling, which is at least more controlled in principle, [this paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3435932/) found the following about general injury risks

> A total of 7 fractures and 8 dislocations were recorded during the study. The incidence of fractures and dislocations was 1.1 per 10 000 athlete exposures, with a fracture rate of 0.5 and a dislocation rate of 0.6

For comparison, [this study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10588583) on professional football (gridiron) players found 

> A total of 108 players sustained fractures during 638,247 hours of player exposure (88.9% training and 11.1% matches), representing 2.7% of all time-loss injuries. The incidence was 0.17 fractures per 1000 h of exposure (match and training incidence of 0.9 and 0.07 fractures / 1000 h, respectively), equivalent to an average of one fracture per team per season.

While these are general risks, it is quite routine in either sport to receive impacts (usually same "z-level) in the kilojoules. It is not clear why the risk of damage should be orders of magnitude greater on the vertical axis rather than the horizontal axis at the same short distance, nor is that really observable. 

**What is the bottom line?** A moderate-sized object falling onto the player from one Z-level should have a moderate chance to cause pain, a small chance to cause token damage, and a very marginal chance to cause double-digit damage.

Suffering major injury in the 30-damage range from a falling wasp should be an occurrence on the order of 0.001% probability or something like that. 

A player character who receives such impacts daily might fear systemic health effects in the future, but nothing that is simulated at the level of gameplay. Indeed, players can shrink or grow multiple times their original size using mutagen and purifiers, so the health implications of mild blunt trauma are nothing to think about from a design perspective.

### Describe alternatives you have considered.

Maybe flesh out the case of objects falling onto feet, which are specially vulnerable to almost any weight from almost any height (and hands if prone?). 

I've still got a subungual hematoma two months after dropping a ~1 kg jar on my (uncovered) big toe. 30 bash damage for stopping a falling 87.5 kg (actually 81.5 kg) object the size of an adult human with a foot sounds reasonable. 

### Additional context

Bemused compliance.

**Comment by PatrikLundell** [comment]
The bizarre spectacle Americans call "wrestling" has little to do with falling objects or hostile attacks. The actors should cooperate to make everything seem as spectacular as possible while actively trying to avoid actual injury (assuming they act in good faith, which apparently isn't always the case). This means that is should look like the "attacker" performs direct attacks on the "victim", but if done correctly most of the force should be taken up by the "attacker" landing beside the "victim". Also, the telegraphing allows the "victim" and the "attacker" to coordinate their actions to distribute the force of the "attack".

The reason your toe is still suffering is because you're not blob infected and thus don't have blob magic healing.

**Comment by Montimorency** [comment]
Absolutely. But in the wrestling spectacle, significant force in kilojoules or pascals of heavy objects (e.g. a leaping wrestler's torso) landing onto body parts from meters above, or just the attacker's own body parts making contact with the ring from the fall, is unavoidable despite calculated mitigation, and that is what is at question in this game mechanic. As I said, there are many real-world approaches to this, and one of them is the - not recommended - phenomenon of catching a falling person. Even being struck on the arms, a relatively-vulnerable body part, by an object the size and density of an adult male human from a couple of meters is unlikely to cause injury. There are also many examples in popular culture of small children being caught from great heights with no ill effect to the catcher, although of course those are much lighter than sub-200 lb objects. 

[Here is an example](https://www.youtube.com/watch?v=n8wHITvWDQU&t=1s) of a man attempting to catch an adult woman, let's assume at least 40 kg, from 11 stories. She dies, and he is hospitalized with "[k]nee joint injuries and tibial fractures, along with several ligament rupture". Why are survivors taking taking that kind of damage - Blob-mediated - from a single story, adjusting for weight and the fact that z-levels are smaller than real-world stories? 

I cannot recommend anyone test these things out to their own satisfaction, of course. Real variability makes any falling or catching inherently more dangerous than it is in a videogame with superhealing. 

One way to expand the game mechanic, at least, would be with pinning underneath a large-enough object as possible outcome, I suppose 'off the shelf' with a slip/knockdown effect. That and an increased risk of objects landing on feet relative to other body parts would help the negation of damage from other circumstances feel more balanced.

**Comment by ADekema** [comment]
Isn´t this just a physics problem. Why not use the the formula for falling objects to conculate kinetic energy?

**Comment by Montimorency** [comment]
The formula was already based on an interpretation of kinetic energy, as is the new one. But damage is still too high.

**Comment by Montimorency** [comment]
I'm just saying that the "realism" factor attests strongly against an average American man fall on you from 2 or 3 meters relative elevation having a similar effect to being shot by 9mm. 

Pain, stunning, or pinning should be much more common than damage.

**Comment by kevingranade** [comment]
Your thought of "maybe falling damage can be further improved" is valid, but nothing from professional wrestling is applicable here. The entire wrestling ring is a safety device to cushion falls, the wrestlers are cooperating to mitigate impact, they are very much NOT landing on each other with full force.

**Comment by Montimorency** [comment]
>they are very much NOT landing on each other with full force.

I agree! But they are falling on each other with the force of gravity, as falling objects (such as dead giants wasps) generally do.

**Comment by Montimorency** [comment]
Just for interest, [one of the various videos](https://www.youtube.com/watch?v=uSQAMx9Bxis 
) related to 'fall damage' I trawled, of a window AC unit landing on a firefighter's head from the second story. 

I'm not sure how to estimate the weight of the thing from perceived size, but it should have been 50-100 lb, a rather dense object, with many irregular, perhaps sharp, metal features. The firefighter was wearing a helmet, so the only effect was to leave him knocked out for a little while, with some damage to the helmet. 

Still not as bad as 9mm to the gut, eh?
